kpet/data.py

```diff
@@ -387,26 +388,105 @@ def validate_host_type_regex(self):
                                   host_type_regex.pattern,
                                   host_types)
 
+    def convert_tree_arches(self):
+        """
+        Get a list of available arches for each tree
+        Returns:
+            Raises a schema.Invalid exception when finding an invalid regex
+        """
+
+        trees = {}
+        # Create a dict {tree: arches,} where arches is a list of regexes
+        for tree in self.trees:
+            trees[tree] = []
+            if "arches" not in self.trees[tree]:
+                self.trees[tree]["arches"] = [".*"]
+
+            for arch_regex in self.trees[tree]["arches"]:
+                temp_trees = []
```

Comment:
`temp_trees` is misguiding. Something like `regex_arhes` would be better. You don't really need this line, though. See below.